pkg/scheduler/plugins/elasticquota/core/runtime_quota_calculator.go

```diff
@@ -48,17 +50,21 @@ func NewQuotaNode(quotaName string, sharedWeight, request, min int64, allowLentR
 // quotaTree abstract the struct to calculate each resource dimension's runtime Quota independently
 type quotaTree struct {
 	quotaNodes map[string]*quotaNode
```

Comment:
Can treemap help in this scenario?

Reply:
there need a simple sort struct  is ok,  this only have insert method